packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -206,7 +210,14 @@ class FileCollection extends DocumentCollection {
       throw new Error('missing name argument')
     }
 
-    const path = uri`/files/${fileId}?Name=${name}&Type=file&Executable=${executable}`
+    let metadataId = ''
```

Comment:
```suggestion
    let metadataId
```

It does not seem necessary to initialize to empty string